uint32[]   pids
uint16[]   nice
uint16[]   num_threads
string[]   name
string[]   exe
string[]   cwd
string[]   status
string[]   username
string[]   terminal
float32[]  create_time
float32[]  memory_percent
float32[]  cpu_percent
